leftwm-core/src/display_servers/xlib_display_server/event_translate_property_notify.rs

```diff
@@ -30,16 +30,10 @@ pub fn from_event(xw: &XWrap, event: xlib::XPropertyEvent) -> Option<DisplayEven
         xlib::XA_WM_NORMAL_HINTS => {
             build_change_for_size_hints(xw, event.window).map(DisplayEvent::WindowChange)
         }
-        xlib::XA_WM_HINTS => match xw.get_wmhints(event.window) {
-            Some(hints) if hints.flags & xlib::InputHint != 0 => {
-                let handle = event.window.into();
-                let mut change = WindowChange::new(handle);
-                change.never_focus = Some(hints.input == 0);
-                Some(DisplayEvent::WindowChange(change))
```

Comment:
The old code would never update never_focus if the window lost the InputHint. I've fixed this in this PR too.